Makefile

```diff
@@ -3,11 +3,11 @@ SOURCES = $(shell find . -name "*.py")
 install:
 	pip install -r requirements.txt
 
-install-dev: install
-	pip install -e ".[dev]"
+install-dev:
+	pip install -r requirements.txt -r requirements-dev.txt .
 
 install-docs:
-	pip install -e ".[dev,docs]"
+	pip install -r requirements.txt -r requirements-docs.txt .
```

Comment:
Does install docs not require the dev install anymore?

Reply:
it does not